src/MenuRegistration/PragmaMenuAndShortcutRegistrationItem.class.st

```diff
@@ -13,6 +13,7 @@ Class {
 		'label',
 		'help',
 		'icon',
```

Comment:
```suggestion
```

Reply:
This instance variable is not used anymore and should be removed

Reply:
Forgot to actually remove it indeed, fixed in commit 49a7a887efa18e50.